sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -501,4 +503,54 @@ export class StudentController extends BaseController {
       restrictionMessage: studentRestrictionStatus.restrictionMessage,
     } as StudentRestrictionDTO;
   }
+
+  /**
+   * API to fetch student details by studentId.
+   * This API will be used by ministry users.
+   * @param studentId
+   * @returns Student Details
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
```

Comment:
Please add the group validation also for AEST.
```
@Groups(UserGroups.AESTUser)
```

Reply:
Feedback comment addressed.